packages/integration-tests/package.json

```diff
@@ -15,6 +16,7 @@
         "webpack": "^5.70.0"
     },
     "devDependencies": {
+        "@trezor/utils": "1.0.1",
```

Comment:
@trezor/transport probably should be listed here too (protobuf.d.ts used in fixtures ? )